energy/energy.ino

```diff
@@ -44,12 +49,13 @@ void setup()
     notecard.begin();
 #endif
 
-    // Set the product UID and set the sync mode to be "minimum"
+    // Set the product UID if defined and set the sync mode to be "minimum"
     J *req = notecard.newRequest("hub.set");
-    JAddStringToObject(req, "product", myProductID);
-    JAddStringToObject(req, "mode", "minimum");
-    notecard.sendRequest(req);
-
+	if (myProductID[0]) {
```

Comment:
The code looks great, but this appears to be overly indented on GitHub.

Actually the whole file looks to be indented strangely.

Can you use VSCode to normalize it one way or the other?